src/Book/CreateOrEditBook.tsx

```diff
@@ -140,6 +145,15 @@ const CreateOrEditBook = (props: Props) => {
         )
     }
 
+    // let once = 1;
+    //
+    // if(props.actDeactCopyError && once === 1){
+    //     notifyError("No se puede cambiar el estado del ejemplar porque se encuentra reservado");
+    //     ++once;
+    //
+    // }
```

Comment:
Try to avoid commenting code that won't be used. Commented code should only be left if it will be used in the future.